File: node/deleteUser.js
const db = require('./utils/dbClient');

exports.handler = async (event) => {
  const { userId } = event.pathParameters;

  await db.delete({
    TableName: 'Users',
    Key: { userId }
  }).promise();

  return {
    statusCode: 204,
    body: ''
  };
};

File: python/delete_user.py
import json
import boto3

dynamodb = boto3.resource('dynamodb')
table = dynamodb.Table('Users')

def lambda_handler(event, context):
    user_id = event['pathParameters']['userId']
    table.delete_item(Key={'userId': user_id})
    
    return {"statusCode": 204}
